libcalico-go/lib/backend/model/tier.go

```diff
@@ -88,5 +88,6 @@ func (options TierListOptions) KeyFromDefaultPath(path string) Key {
 }
 
 type Tier struct {
-	Order *float64 `json:"order,omitempty"`
+	Order         *float64 `json:"order,omitempty"`
+	DefaultAction string   `json:"defaultAction,omitempty"`
```

Comment:
Perhaps I misinterpreted Shaun's comment, but I think he meant for `DefaultAction` to be a `v3.Action` type and not strictly a `string` type on the model API.

>  I prefer to keep all new additions to the model API in-sync with the v3 API to avoid surprises

I think the main thing was that it shouldn't be a `bool`